src/scripts/clipperUI/panels/signInPanel.tsx

```diff
@@ -60,52 +60,84 @@ class SignInPanelClass extends ComponentBase<SignInPanelState, SignInPanelProps>
 			&& this.props.clipperState.userResult.data.errorDescription.indexOf("OrgId") === 0;
 	}
 
-	errorDescriptionControlHandler() {
-		this.setState({ errorDescriptionShowing: !this.state.errorDescriptionShowing });
+	signInFailureThirdPartyCookiesBlocked(): boolean {
+		return !!this.props.clipperState.userResult && !!this.props.clipperState.userResult.data
+			&& this.props.clipperState.userResult.data.updateReason === UpdateReason.SignInAttempt
+			&& !this.props.clipperState.userResult.data.writeableCookies;
+	}
+
+	debugInformationControlHandler() {
+		this.setState({ debugInformationShowing: !this.state.debugInformationShowing });
+	}
+
+	signInErrorDetected() {
+		return this.signInFailureContainsErrorDescription() || this.signInFailureThirdPartyCookiesBlocked();
 	}
 
-	errorInformationToggle() {
-		if (this.getSignInFailureDetected()) {
-			return <div className="signInErrorToggleInformation">
-				<a id={Constants.Ids.signInErrorMoreInformation} {...this.enableInvoke(this.errorDescriptionControlHandler, 10) }>
+	errorMoreInformationTogggle() {
+		if (this.signInErrorDetected()) {
+			return <div id="signInErrorToggleInformation">
+				<a id={Constants.Ids.signInErrorMoreInformation} {...this.enableInvoke(this.debugInformationControlHandler, 10) }>
 					<img id={Constants.Ids.signInToggleErrorDropdownArrow} src={ExtensionUtils.getImageResourceUrl("dropdown_arrow.png")} />
 					<span id={Constants.Ids.signInToggleErrorInformationText}
 						style={Localization.getFontFamilyAsStyle(Localization.FontFamily.Light)}>
-						{this.state.errorDescriptionShowing
+						{this.state.debugInformationShowing
```

Comment:
I'm going to hold off on this one for now.  I can see this being something we use in other places at some point, and so I'll wait until then to do the refactoring.